src/coverlet.msbuild.tasks/MSBuildLogger.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using ILogger = Coverlet.Core.Logging.ILogger;
+
+namespace Coverlet.MSbuild.Tasks
+{
+    class MSBuildLogger : ILogger
+    {
+        private readonly TaskLoggingHelper _log;
+
+        public MSBuildLogger(TaskLoggingHelper log)
```

Comment:
you can simplify also constructor `public MSBuildLogger(TaskLogginHelper log) => _log = log;`